### DISCUSSION TITLE
New vignette

### DISCUSSION THREAD

**Discussion by rcatlord** [tech-acs/crvsreportpackage#28] [Announcements]
Hi @beardl-ons, @diego-ons, @gulilattesfaye, @jeroenminderman, @RachelS-ONS, and @tbelay,

I have created a [new vignette](https://tech-acs.github.io/crvsreportpackage/articles/table_templates.html) showing all of the tables and variables needed in a CRVS report. You can navigate between all of the 59 tables using the table of contents. You can also share a link to a particular table (e.g. [Table 4.3](https://tech-acs.github.io/crvsreportpackage/articles/table_templates.html#table-4-3)).

Please post any comments in the thread below.

Thanks,

Henry

**Comment by diego-ons** [comment]
Looks really good and have been using it so far to change some of the names of the functions! Really useful!

**Comment by jeroenminderman** [comment]
Agreed super comprehensive! Thanks Henry.
